docs/static/settings-file.asciidoc

```diff
@@ -28,4 +29,82 @@ pipeline.batch.size: 125
 pipeline.batch.delay: 5
 -------------------------------------------------------------------------------------
 
-See <<command-line-flags>> for a description of the available options.
+The `logstash.yml` file includes the following settings:
+
+ *`node.name`*::
```

Comment:
Can we convert this into a table with 3 columns: `setting type`, `setting definition` and `default value`